modules/apps/27-interchain-accounts/controller/keeper/relay.go

```diff
@@ -16,7 +16,7 @@ import (
 // absolute timeoutTimestamp must be provided. If the packet is timed out, the channel will be closed.
 // In the case of channel closure, a new channel may be reopened to reconnect to the host chain.
 func (k Keeper) SendTx(ctx sdk.Context, chanCap *capabilitytypes.Capability, connectionID, portID string, icaPacketData icatypes.InterchainAccountPacketData, timeoutTimestamp uint64) (uint64, error) {
-	activeChannelID, found := k.GetActiveChannelID(ctx, connectionID, portID)
+	activeChannelID, found := k.GetOpenActiveChannel(ctx, connectionID, portID)
```

Comment:
`GetActiveChannelID` is no longer used in both controller and host submodules. All previous calls are now using `GetOpenActiveChannel`

Reply:
I vote to still leave the helper fn in for the time being. Having standard getters/setters around sometimes is no harm.

Reply:
I agree with @seantking. If the active channel closes, it'd be nice to have a way to get the channel id

Reply:
Yeah sure, I agree in principle. My only nit is just: is it really an "active channel" if it's not in an `OPEN` state? When I think "active channel" its almost implicit in my mind that it's `OPEN`, but maybe just me! My only fear is that it may be a little misleading

Reply:
I agree it can be misleading, but then we need to change the naming of the mapping. I think we are getting into over engineering territory. Hopefully we will delete all of this within a year

Reply:
Agree++